netutils/netinit/netinit.c

```diff
@@ -62,6 +49,10 @@
 
 #include <nuttx/net/mii.h>
 
+#if defined(CONFIG_FSUTILS_IPCFG)
+#  include <fsutils/ipcfg.h>
+#endif
```

Comment:
Technically, should be enclosed in quotes because this is not a system include.

Reply:
@patacongo - good feedback. Thank you.